### ISSUE TITLE
Current version of psycopg (2.6.1) in requirements.txt  incompatible with PostgreSQL 10

### ISSUE THREAD

**Issue by jl24** [CiviWiki/OpenCiviWiki#188]
The current psycopg version referenced in our requirements.txt file is unable to determine the PostgreSQL version properly. This prevents proper installation of the library.

Error Log:
```
Collecting psycopg2==2.6.1 (from -r requirements.txt (line 45))
  Downloading https://files.pythonhosted.org/packages/86/fd/cc8315be63a41fe000cce20482a917e874cdc1151e62cb0141f5e55f711e/psycopg2-2.6.1.tar.gz (371kB)
    100% |████████████████████████████████| 378kB 333kB/s
    Complete output from command python setup.py egg_info:
    running egg_info
    creating pip-egg-info/psycopg2.egg-info
    writing pip-egg-info/psycopg2.egg-info/PKG-INFO
    writing dependency_links to pip-egg-info/psycopg2.egg-info/dependency_links.txt
    writing top-level names to pip-egg-info/psycopg2.egg-info/top_level.txt
    writing manifest file 'pip-egg-info/psycopg2.egg-info/SOURCES.txt'
    warning: manifest_maker: standard file '-c' not found

    Error: could not determine PostgreSQL version from '10.2'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/07/c09gy11n0jg5mpyzvtqr05sh0000gn/T/pip-build-95naqyvp/psycopg2/
```

## Solution
The issue has been fixed in psycopg versions >= 2.7 ([github issue](https://github.com/psycopg/psycopg2/issues/489)), so we should upgrade to the latest release of psycopg (2.7.4).

**Comment by GokuMizuno** [comment]
Fixing this should also fix issue #65

**Comment by brylie** [comment]
Moving to 0.2, since related issue is also in 0.2 milestone.
